x-pack/plugins/security_solution/public/common/components/ml/influencers/get_user_name_from_influencers.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { cloneDeep } from 'lodash/fp';
+import { getUserNameFromInfluencers } from './get_user_name_from_influencers';
+import { mockAnomalies } from '../mock';
+
+describe('get_user_name_from_influencers', () => {
+  let anomalies = cloneDeep(mockAnomalies);
+
+  beforeEach(() => {
+    anomalies = cloneDeep(mockAnomalies);
+  });
+
+  test('returns user names from influencers from the mock', () => {
+    const userName = getUserNameFromInfluencers(anomalies.anomalies[0].influencers);
+    expect(userName).toEqual('root');
+  });
+
+  test('returns null if there are no influencers from the mock', () => {
+    anomalies.anomalies[0].influencers = [];
+    const userName = getUserNameFromInfluencers(anomalies.anomalies[0].influencers);
```

Comment:
```suggestion
    const userName = getUserNameFromInfluencers([]);
```
nit for all tests here, why even set that variable? or at least don't reassign that property, make a `const`

Reply:
Wow, that is dumb. Fixing it!